app/views/comments/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<div>
+    <h1>Add new comment</h1>
+    <% form_with model: @comment, url: user_post_comments_path(user_id: @post.author_id, post_id: @post.id) do |f| %>
+        <%= f.label :text %><br>
+        <%= f.text_area :text, cols: 30, rows: 10 %><br>
+        <%= f.submit "Submit", class: "form_btn" %>
+    <% end %>
+</div>
```

Comment:
- [ ] ı cant reaches this page because of the post error. 

_Next code reviewer, kindly checks if the form working without error or not._